src/main/java/io/harness/cf/client/api/CfClient.java

```diff
@@ -76,7 +79,7 @@ public void init() throws ApiException, CfClientException {
 
     initCache(environmentID);
     if (!config.isStreamEnabled()) {
-      poller.startAsync();
+      startPollingMode();
```

Comment:
It looks like when stream mode is disabled, poller is not initialised, so this causes an error.  I've called 'startPollingMode' here which initialises poller and calls startAsync